xml/Menu/civalpa.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<menu>
+  <item>
+    <path>civicrm/admin/civalpa</path>
+    <page_callback>CRM_Civalpa_Form_CivalpaSettings</page_callback>
+    <title>CivalpaSettings</title>
+    <access_arguments>access CiviCRM</access_arguments>
```

Comment:
possibly add `administer CiviCRM` also?

The `access CiviCRM` is a very broad permission, and the Administer menu should be accessed by Administrators and power users.